server/models/address.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import "time"
+
+type Address struct {
+	Address                      string    `json:"address" bson:"address"`
+	Owner                        string    `json:"owner" bson:"owner"`
+	Balance                      string    `json:"balance" bson:"balance"`
+	BalanceInt                   int64     `json:"balance_int" bson:"balance_int"`
+	LastUpdatedAt                time.Time `json:"last_updated_at" bson:"last_updated_at"`
+	TokenName                    string    `json:"token_name" bson:"token_name"`
```

Comment:
Adding the `omitempty` json option to many of these fields would clean up and reduce the size of the response. Alternatively, they could be grouped into an object on a single field that would be null or omitted.